test/built-ins/Infinity/S15.1.1.2_A3_T1.js

```diff
@@ -5,14 +5,14 @@
 info: The Infinity is DontDelete
 es5id: 15.1.1.2_A3_T1
 description: Use delete
-includes: [propertyHelper.js, fnGlobalObject.js]
+includes: [propertyHelper.js]
 ---*/
 
 // CHECK#1
-verifyNotConfigurable(fnGlobalObject(), "Infinity");
+verifyNotConfigurable(this, "Infinity");
 
 try {
-  if (delete fnGlobalObject().Infinity !== false) {
+  if (delete this.Infinity !== false) {
```

Comment:
this is another off-topic. This case is unnecessary, already covered at line 12 with verifyNotConfigurable